src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appId__/components/adaptiveForm/formcontainer/.content.xml

```diff
@@ -5,4 +5,8 @@
     jcr:title="Adaptive Form Container"
     fd:version="2.1"
     sling:resourceSuperType="core/fd/components/form/container/v2/container"
-    componentGroup=".${appTitle} - Adaptive Form"/>
+  #if($isFormsSpa)
+    componentGroup="${appTitle} - Adaptive Form"/>
+  #else
+  componentGroup=".${appTitle} - Adaptive Form"/>
```

Comment:
We would expose form container in march release

Reply:
So should we keep it hidden in current release? How will the customer use it?